Broken master build

### Affected Version

After #6953 

### Description

Build with `-Pdist` flag fails with the below error.

```
ERROR StatusLogger No log4j2 configuration file found. Using default configuration: logging only errors to the console.
Mar 27, 2019 12:03:32 PM org.hibernate.validator.internal.util.Version <clinit>
INFO: HV000001: Hibernate Validator 5.1.3.Final
12:03:34.459 [main] ERROR org.apache.druid.cli.PullDependencies - Unable to resolve artifacts for [org.apache.druid.extensions:druid-google-extensions:jar:0.15.0-incubating-SNAPSHOT (runtime) -> [] < [ (https://repo1.maven.org/maven2/, releases+snapshots)]].
org.eclipse.aether.resolution.DependencyResolutionException: Could not find artifact org.apache.druid.extensions:druid-google-extensions:jar:0.15.0-incubating-SNAPSHOT in  (https://repo1.maven.org/maven2/)
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveDependencies(DefaultRepositorySystem.java:384) ~[aether-impl-0.9.0.M2.jar:?]
	at io.tesla.aether.internal.DefaultTeslaAether.resolveArtifacts(DefaultTeslaAether.java:289) ~[tesla-aether-0.0.5.jar:0.0.5]
	at org.apache.druid.cli.PullDependencies.downloadExtension(PullDependencies.java:400) [druid-services-0.15.0-incubating-SNAPSHOT.jar:0.15.0-incubating-SNAPSHOT]
	at org.apache.druid.cli.PullDependencies.run(PullDependencies.java:299) [druid-services-0.15.0-incubating-SNAPSHOT.jar:0.15.0-incubating-SNAPSHOT]
Exception in thread "main" java.lang.RuntimeException: java.lang.RuntimeException: org.eclipse.aether.resolution.DependencyResol	at org.apache.druid.cli.Main.main(Main.java:117) [druid-services-0.15.0-incubating-SNAPSHOT.jar:0.15.0-incubating-SNAPSHOT]
utionException: Could not find artifact org.apache.druid.extensions:druid-google-extensions:jar:0.15.0-incubating-SNAPSHOT in  (https://repo1.maven.org/maven2/)
Caused by: org.eclipse.aether.resolution.ArtifactResolutionException: Could not find artifact org.apache.druid.extensions:druid-	at org.apache.druid.cli.PullDependencies.run(PullDependencies.java:323)
	at org.apache.druid.cli.Main.main(Main.java:117)
google-extensions:jar:0.15.0-incubating-SNAPSHOT in  (https://repo1.maven.org/maven2/)
Caused by: java.lang.RuntimeException: org.eclipse.aether.resolution.DependencyResolutionException: Could not find artifact org.	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:459) ~[aether-impl-0.9.0.M2.jar:?]
apache.druid.extensions:druid-google-extensions:jar:0.15.0-incubating-SNAPSHOT in  (https://repo1.maven.org/maven2/)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifacts(DefaultArtifactResolver.java:262) ~[aether-impl-0.	at org.apache.druid.cli.PullDependencies.downloadExtension(PullDependencies.java:413)
9.0.M2.jar:?]
	at org.apache.druid.cli.PullDependencies.run(PullDependencies.java:299)
	... 1 more
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveDependencies(DefaultRepositorySystem.java:367) ~[aether-impl-0.9.0.M2.jar:?]
Caused by: org.eclipse.aether.resolution.DependencyResolutionException: Could not find artifact org.apache.druid.extensions:drui	... 4 more
d-google-extensions:jar:0.15.0-incubating-SNAPSHOT in  (https://repo1.maven.org/maven2/)
Caused by: org.eclipse.aether.transfer.ArtifactNotFoundException: Could not find artifact org.apache.druid.extensions:druid-google-extensions:jar:0.15.0-incubating-SNAPSHOT in  (https://repo1.maven.org/maven2/)
	at io.tesla.aether.connector.AetherRepositoryConnector$2.wrap(AetherRepositoryConnector.java:828) ~[aether-connector	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveDependencies(DefaultRepositorySystem.java:384)
-okhttp-0.0.9.jar:0.0.9]
	at io.tesla.aether.internal.DefaultTeslaAether.resolveArtifacts(DefaultTeslaAether.java:289)
	at org.apache.druid.cli.PullDependencies.downloadExtension(PullDependencies.java:400)
	... 2 more
	at io.tesla.aether.connector.AetherRepositoryConnector$2.wrap(AetherRepositoryConnector.java:824) ~[aether-connector-okhttp-0.0.9.jar:0.0.9]
Caused by: org.eclipse.aether.resolution.ArtifactResolutionException: Could not find artifact org.apache.druid.extensions:druid-google-extensions:jar:0.15.0-incubating-SNAPSHOT in  (https://repo1.maven.org/maven2/)
	at io.tesla.aether.connector.AetherRepositoryConnector$GetTask.flush(AetherRepositoryConnector.java:619) ~[aether-connector-okh	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:459)
ttp-0.0.9.jar:0.0.9]
	at io.tesla.aether.connector.AetherRepositoryConnector.get(AetherRepositoryConnector.java:238) ~[aether-connector-okhttp-0.0.9.jar:0.0.9]
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifacts(DefaultArtifactResolver.java:262)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.performDownloads(DefaultArtifactResolver.java:535) ~[aether-impl-0.9.0.M2.jar:?]
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveDependencies(DefaultRepositorySystem.java:367)
	... 4 more
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:436) ~[aether-impl-0.9.0.M2.jar:?]
Caused by: org.eclipse.aether.transfer.ArtifactNotFoundException: Could not find artifact org.apache.druid.extensions:druid-google-extensions:jar:0.15.0-incubating-SNAPSHOT in  (https://repo1.maven.org/maven2/)
	at io.tesla.aether.connector.AetherRepositoryConnector$2.wrap(AetherRepositoryConnector.java:828)
	at io.tesla.aether.connector.AetherRepositoryConnector$2.wrap(AetherRepositoryConnector.java:824)
	at io.tesla.aether.connector.AetherRepositoryConnector$GetTask.flush(AetherRepositoryConnector.java:619)
	at io.tesla.aether.connector.AetherRepositoryConnector.get(AetherRepositoryConnector.java:238)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.performDownloads(DefaultArtifactResolver.java:535)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:436)
	... 6 more
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifacts(DefaultArtifactResolver.java:262) ~[aether-impl-0.9.0.M2.jar:?]
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveDependencies(DefaultRepositorySystem.java:367) ~[aether-impl-0.9.0.M2.jar:?]
	... 4 more
```

--- a/extensions-core/google-extensions/pom.xml
+++ b/extensions-core/google-extensions/pom.xml
@@ -21,7 +21,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.apache.druid.extensions.contrib</groupId>
+    <groupId>org.apache.druid.extensions</groupId>
     <artifactId>druid-google-extensions</artifactId>
     <name>druid-google-extensions</name>
     <description>druid-google-extensions</description>

